src/domain/community/user/user.entity.ts

```diff
@@ -25,22 +26,22 @@ export class User extends ContributorBase implements IUser {
   rowId!: number;
 
   @Column()
```

Comment:
All these columns really should also have lenghts / nullable false

Reply:
Let's discuss them additionally. They are currently `varchar(255)`.